packages/back-end/src/controllers/features.ts

```diff
@@ -974,7 +1063,13 @@ async function getDraftRevision(
   if (!revision) {
     throw new Error("Cannot find revision");
   }
-  if (revision.status !== "draft") {
+  if (
+    !(
+      revision.status === "draft" ||
+      revision.status === "pending-review" ||
+      revision.status === "reviewed"
```

Comment:
Are these statuses correct?  From the typedef, it looks like `reviewed` is not one of the options, but `approved` and `changes-requested` are.